Scripts/Runtime/Data/Collections/UnsafeTypedStream.cs

```diff
@@ -400,10 +473,18 @@ public LaneWriter AsLaneWriter(int laneIndex)
             [NativeDisableUnsafePtrRestriction] private readonly BufferInfo* m_BufferInfo;
             [NativeDisableUnsafePtrRestriction] private readonly LaneInfo* m_Lane;
 
-            internal LaneWriter(LaneInfo* lane, BufferInfo* bufferInfo)
+            /// <summary>
+            /// Reports if this <see cref="LaneWriter"/> has been created and is valid to use.
+            /// </summary>
+            public bool IsCreated
+            {
+                get => m_BufferInfo != null;
```

Comment:
Some comment as on `Writer` about isCreated. Although in this case you'll get the error when the constructor tries to populate `m_Lane`

Reply:
I made sure these constructors can't be called without going through an `IsCreated` assert first now.